src/Events/Services/InboundService.cs

```diff
@@ -48,17 +48,45 @@ public class AppEventsService : IAppEventsService
         }
 
         /// <inheritdoc/>
-        public async Task<string> StoreCloudEvent(CloudEvent cloudEvent)
+        public async Task<string> Save(CloudEvent cloudEvent)
```

Comment:
Is the need for SaveAndPostInbound only to support the old app-event implementation, or will we use it going forward? Should we document the difference in functionality (that SaveAndPost creates the id) in the function documenation?